tests/test_arch_cuda_image.py

```
@@ -82,11 +82,24 @@ def test_validate_script_has_off_gpu_and_required_gpu_modes(self):
+                "ls-files",
+                "--cached",
+                "--others",
+                "--exclude-standard",
+                "*.zsh",
+            ],
+            cwd=REPO_ROOT,
+            text=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            check=True,
+        ).stdout.splitlines()
```

Ruff (0.16.1)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)